spec/models/manageiq/providers/kubernetes/container_manager/event_catcher_mixin_spec.rb

```diff
@@ -279,5 +279,101 @@ def initialize(ems = nil)
         end
       end
     end
+
+    context 'given an event with an unsupported kind' do
+      let(:kubernetes_event) do
+        {
+          'reason'         => 'DoesItReallyMatter',
+          'involvedObject' => {
+            'kind' => 'SomeRandomObject',
+          },
+        }
+      end
+
+      it 'will return nil' do
+        event = RecursiveOpenStruct.new(:object => kubernetes_event)
+        expect(test_class.new.extract_event_data(event)).to be nil
+      end
+    end
+
+    context 'given an event with an unsupported reason' do
+      let(:kubernetes_event) do
+        {
+          'reason'         => 'DoesItReallyMatter',
+          'involvedObject' => {
+            'kind' => 'ReplicationController',
+          },
+        }
+      end
+
+      it 'will return nil' do
+        event = RecursiveOpenStruct.new(:object => kubernetes_event)
+        expect(test_class.new.extract_event_data(event)).to be nil
+      end
+    end
+  end
+
+  describe "#filtered?" do
+    let(:kubernetes_event) do
+      {
+        'metadata'       => {
+          'name'              => 'mysql-1.146486622e01d244',
+          'namespace'         => 'proj',
+          'selfLink'          => '/api/v1/namespaces/proj/events/mysql-1.146486622e01d244',
+          'uid'               => '4c513e6d-525d-11e6-8564-525400c7c086',
+          'resourceVersion'   => '1360577',
+          'creationTimestamp' => '2016-07-25T11:45:34Z',
+          'deletionTimestamp' => '2016-07-25T13:45:34Z'
+        },
+        'involvedObject' => {
+          'kind'            => 'ReplicationController',
+          'namespace'       => 'proj',
+          'name'            => 'mysql-1',
+          'uid'             => '7599d451-4c1c-11e6-89dd-525400c7c086',
+          'apiVersion'      => 'v1',
+          'resourceVersion' => '1360571'
+        },
+        'reason'         => 'SuccessfulCreate',
+        'message'        => 'Created pod: mysql-1-i4b54',
+        'source'         => {
+          'component' => 'replication-controller'
+        },
+        'firstTimestamp' => '2016-07-25T11:45:34Z',
+        'lastTimestamp'  => '2016-07-25T11:45:34Z',
+        'count'          => 1,
+        'type'           => 'Normal'
+      }
+    end
+
+    let(:event) { RecursiveOpenStruct.new(:object => kubernetes_event) }
+
+    let(:test_instance) do
+      test_class.new.tap do |i|
+        allow(i).to receive(:filtered_events).and_return([])
+      end
+    end
+
+    it 'with a blacklisted event' do
+      expect(test_instance).to receive(:filtered_events).and_return(["REPLICATOR_SUCCESSFULCREATE"])
```

Comment:
Ignore above idea.
Automate is what translates to the lowercase naming scheme, and since we want blacklisted events to have minimal impact, we don't want them triggering Automate.
Filtering here must use the uppercase names.